src/components/App/AppData.js

```diff
@@ -17,6 +18,15 @@ function AppData() {
     mySitesData,
   } = useMySites()
 
+  const sites = useSites()
+
+  function features(site) {
+    return site.enabledFeatures.find(
+      (feature) => feature.key.toLowerCase() === 'has_app',
+    )
+  }
+  const hasApp = sites?.data?.results?.filter(features)
```

Comment:
Variable names starting with `has` or `is` are normally reserved for variables with a simple boolean value. Given that the value of `hasApp` is an array of sites, please rename it to avoid confusion e.g. `sitesWithApps`